.github/workflows/ci.yml

```diff
@@ -24,7 +24,31 @@ env:
   LOCALE_ID: 'en-US'
 
 jobs:
+  check-run:
+    runs-on: 'ubuntu-latest'
+    outputs:
+      should-skip: ${{ steps.check.outputs.should-skip }}
+    steps:
+      - name: Check for duplicate push event with PR
+        id: check
+        if: github.event_name == 'push'
+        run: |
+          # Check if this push is tied to an open PR
+          PR_NUMBER=$(curl -s -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+            "https://api.github.com/repos/${{ github.repository }}/commits/${{ github.sha }}/pulls" | \
```

Comment:
So if you're on a fork, it will look for a PR on that fork, right? I think that's probably what makes the most sense.

Reply:
Yes, but if it's annoying, we can fix it later. We need to use it more and see how well it works.